front-end/virtual_pet/lib/pages/minutes.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class MyMinutes extends StatelessWidget {
+  final int mins;
+
+  const MyMinutes({required this.mins, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 5.0),
+      child: Center(
+        child: Text(
+          // mins < 10 ? '0' + mins.toString() : mins.toString(),
```

Comment:
We can remove this line since the updated string interpolation line is right below